src/_selectors/AssetIndexSelectors.js

```diff
@@ -23,7 +24,32 @@ export const shownAssetIndexRowsSelector = createSelector(
     }
 );
 
-export const assetIndexHeadersSelector = createSelector(
+const assetIndexRowToDuration = row =>
+    row ? `${row[2]}–${row[3]}` : '—';
+
+const assetIndexTableSelector = createSelector(
+    shownAssetIndexRowsSelector,
```

Comment:
Yes, I am considering this data transformation layer to be in LiveApi even.
We can add it as LiveDataTransform.js so other can use it too.